helm/examples/pods-list/cloudbuild.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+- name: 'gcr.io/cloud-builders/helm'
+  args: ['list']
+  env:
+  - 'CLOUDSDK_COMPUTE_ZONE=us-east4-b'
```

Comment:
Have you tried this example?

If helm shells out to the available `kubectl` I don't believe this will work (those env vars are interpreted in a shell script wrapper).